requirements/edx-sandbox/py35.txt

```diff
@@ -9,7 +9,7 @@ common/lib/symmath  # via -r requirements/edx-sandbox/py35.in
 git+https://github.com/edx/openedx-calc.git@e9b698c85ad1152002bc0868f475f153dce88952#egg=calc==0.4  # via -r requirements/edx-sandbox/py35.in
 cffi==1.14.0              # via -r requirements/edx-sandbox/shared.txt, cryptography
 git+https://github.com/edx/openedx-chem.git@ff4e3a03d3c7610e47a9af08eb648d8aabe2eb18#egg=chem==1.0.0  # via -r requirements/edx-sandbox/py35.in
-cryptography==2.8         # via -r requirements/edx-sandbox/shared.txt
+cryptography==2.9         # via -r requirements/edx-sandbox/shared.txt
```

Comment:
There are a bunch of backward-incompatible changes here.  Does anyone know off hand if these are all safe, or should we add a constraint while we work that out?
https://github.com/pyca/cryptography/blob/master/CHANGELOG.rst#29---2020-04-02

Reply:
Not sure if bokchoy failure is related.  I think it may not be, but here was the failure:
Run tests / 13 / acceptance.tests.video.test_video_module.YouTubeVideoTest.test_video_component_stores_speed_correctly_for_multiple_videos

Reply:
Guess the bokchoy failure was unrelated.